taglib/src/test/java/com.reevoo.taglib/ReevooTaglibTest.java

```diff
@@ -24,7 +24,7 @@ public void setUp() throws Exception {
         super.setUp();
         reevooTag.setDynamicAttribute("", "sku", "ABC123");
         reevooTag.setTrkref("FOO");
-        reevooTag.setBaseURI("http://mark.reevoo.com/endpoint");
+        reevooTag.setBaseURI("https://mark.reevoo.com/endpoint");
```

Comment:
I think that is a bad idea we should not ever support http. Everything should be https due to security